renderer/views/player.js

```diff
@@ -116,6 +117,7 @@ function renderPlayerControls (state, dispatch) {
 
   // Handles a click or drag to scrub (jump to another position in the video)
   function handleScrub (e) {
+    dispatch('videoMouseMoved')
```

Comment:
I was able to reproduce -- I think once you're dragging it may not be causing the mousemove event to fire.